docs/tutorials/screen-resolution-detection.md

```diff
@@ -32,6 +32,24 @@ Note: in order to use this feature in a UWP app, you must add the URI of your
 web app in the ContentURIs section of the Package.appxmanifest file and set
 the `WinRT access` field to `All`.
 
+### WebView2
+
+When using a WebView2 control in a UWP app, additional steps are required in
+order to enable the screen resolution detection. First, the WebView2's user-agent
+is the same as Edge Chromium and does not contain the term "Xbox One", so it has
+to be manually added like this when initializing your WebView2:
+
+```CSharp
+webView.CoreWebView2.Settings.UserAgent += " Xbox One";
```

Comment:
Is there some improvement we could make to UA detection to understand WebView2 as Xbox One without this?

Reply:
Based only on the user agent, I don't think it will be possible:

Edge Chromium on Windows 11:
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36 Edg/127.0.0.0

WebView2 on Xbox (not modified):
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36 Edg/125.0.0.0

They are basically the same, except for the Chrome/Edge version, which is a bit newer on Windows.


Maybe that detecting the presence of the WinRT APIs could be enough. Something like 

```
static isXboxOne() {
    return shaka.util.Platform.userAgentContains_('Xbox One') || typeof Windows !== 'undefined' || typeof chrome?.webview?.hostObjects?.sync?.Windows !== 'undefined';
}
```

But it could still be a UWP running on Windows... Maybe there's a way to get the device type from WinRT once we can access to it.

Reply:
Okay, thanks for considering it.  We are very open to improvements in detection later.